### PR TITLE
Document FtSecure in README #437

### PR DESCRIPTION
**Pull request by neonailol**
PR for #437

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#783](https://github.com/yegor256/takes/issues/783) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#783](https://github.com/yegor256/takes/issues/783) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: README.md [hunk 1]
```diff
@@ -759,6 +760,30 @@ public final class TkIndex implements Take {
...
+System.setProperty("javax.net.ssl.trustStore", file);
+System.setProperty("javax.net.ssl.trustStorePassword", password);
+```
+
+Then simple create exemplar of class [FtSecure](src/main/java/org/takes/http/FtSecure.java) with socket factory
+
+```java
+final ServerSocket skt = SSLServerSocketFactory.getDefault().createServerSocket(0);
+final FtRemote remote = new FtRemote(
+                            new FtSecure(new BkBasic(new TkFixed("hello, world")), skt),
```
Comment (g4s8) [inline README.md]: @neonailol please, format the code just as all other code fragments in README (2 whitespaces indentation)

**Comment by neonailol** [comment]
@g4s8 formatted code with 2 whitespaces intendation

File: README.md [hunk 2]
```diff
@@ -759,6 +760,29 @@ public final class TkIndex implements Take {
   }
 }
 ```
+## SSL Configuration
+
+First of all, setup your keystore settings, for example
+
+```java
+  final String file = this.getClass().getResource("/org/takes/http/keystore").getFile();
```
Comment (g4s8) [inline README.md]: @neonailol here code block should not be indented

**Comment by neonailol** [comment]
@rultor good to merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8), total is [+3760](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+9365](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#783](https://github.com/yegor256/takes/issues/783) is now out of scope

**Pull request merged**